h/streamer.py

```diff
@@ -464,15 +468,23 @@ def start_reader(cls, request):
             return
         cls.event_queue = gevent.queue.Queue()
         reader_id = 'stream-{}#ephemeral'.format(_random_id())
-        reader = request.get_queue_reader('annotations', reader_id)
-        reader.on_message.connect(cls.on_queue_message)
-        reader.start(block=False)
+
+        for topic in ['annotations', 'user']:
+            reader = request.get_queue_reader(topic, reader_id)
+            reader.on_message.connect(
+                receiver=partial(cls.on_queue_message, topic),
```

Comment:
You don't need to do this. The `gnsq.Reader` instance is already passed into the message handler, and it has a `topic` attribute.